### ISSUE TITLE
No Braille output of Trill line

### ISSUE THREAD

**Issue by jmwmusic** [musescore/MuseScore#34356]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

STEPS:
1.  Add a trill line from the ornaments panel to a note.
2. Notice there is no change to the Braille.  The note still shows as before.  The sign for the trill is not added.

See also #34355 as this is not being output correctly with screen reader speech also.

### Supporting files, videos and screenshots

N/A

### In which versions of MuseScore Studio is this issue present?

4.7.4

### Regression

I was unable to check

### Operating system

Windows 11 with NVDA

### Checklist

- [x] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [x] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [x] I have attached all requested files and information to this report
- [x] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by fcnjd** [comment]
Also fixed in PR #33862 , which initially adds those lines and also adds them to the doubling logic, if more than three in a row are present.
